src/auro-drawer.js

```diff
@@ -25,7 +25,8 @@ const CONFIG = {
  * @slot close - Used to replace the close button.
  *
  * @fires auroDrawer-toggled - Event fired when the drawer is toggled open or closed.
- * 
+ *
+ * @csspart backdrop - to style the backdrop behind the the content wrapper.
```

Comment:
```suggestion
 * @csspart drawer-backdrop - to style the backdrop behind the the content wrapper.
```

Shouldn't this follow the same naming pattern as lines 30-32?